lib/rules/property-no-vendor-prefix/__tests__/index.js

```diff
@@ -100,3 +100,46 @@ testRule(rule, {
     }
   ]
 });
+
+testRule(rule, {
+  ruleName,
+  config: [
+    true,
+    { ignoreProperties: ["transform", "columns", "/^animation-/"] }
+  ],
+
+  accept: [
```

Comment:
Add `a { -wEbKiT-tRaNsFoRm: scale(1); }` `accept` test. That we know it's case insensitive.

Both config value case and property case should be case insensitive.

`ignoreProperties: ["transform"]` should match `a { -webkit-TrAnSfOrM: none; }`. And `ignoreProperties: ["TrAnSfOrM"]` should match `a { -webkit-trAnsfOrm: none; }`, etc.

Could you also add test for the later one, please?

Reply:
> Both config value case and property case should be case insensitive.

I believe this behaviour [was changed in `8.0.0`](https://github.com/stylelint/stylelint/pull/2709). Options are now case sensitive and `optionsMatches` will behave as such. Users should explicitly use the `i` RegExp flag for case-insensitivity e.g. `ignoreProperties: ["/transform/i"]`.

So `ignoreProperties: ["transform"] should not match `a { -webkit-TrAnSfOrM: none; }`.

It might be worth updating the `config` to `{ ignoreProperties: ["transform", "columns", "/^animation-/i"] }` so that this behaviour can be confirmed in this rule i.e. `animation-name` and `ANIMATION-name` will both match, but `TRANSFORM` will not.